tests/testthat/helper-scope.R

```diff
@@ -1,5 +1,16 @@
-renv_tests_scope <- function(packages = character(), project = NULL, envir = parent.frame()) {
 
+renv_tests_scope <- function(packages = character(),
+                             project = NULL,
+                             envir = parent.frame())
+{
+  # source setup.R if necessary (for interactive scenarios)
+  running <- getOption("renv.tests.running", default = FALSE)
```

Comment:
You should be able to revert all these changes now, right?

Reply:
Don't I still need this if I want `setup.R` to be run before running a test?

Reply:
No? It should be run automatically.

Reply:
I'm not seeing that on main right now (unless I keep in the bit of code referenced here).